Wikipedia/Code/TalkPageTopicListViewController.swift

```diff
@@ -241,11 +241,12 @@ private extension TalkPageTopicListViewController {
     }
     
     func configure(cell: TalkPageTopicCell, at indexPath: IndexPath) {
-        guard let title = fetchedResultsController.object(at: indexPath).title else {
+        let topic = fetchedResultsController.object(at: indexPath)
+        guard let title = topic.title else {
             return
         }
         
-        cell.configure(title: title)
+        cell.configure(title: title, isRead: topic.isRead)
```

Comment:
Another call to this configure method happens when the cell is updated -

https://github.com/wikimedia/wikipedia-ios/blob/unread/Wikipedia/Code/TalkPageReplyListViewController.swift#L263

should `isRead` bet set the same way there instead of defaulting to `false`?

Reply:
That one is for `TalkPageReply`s which IIRC we aren't supposed to handle unread state for yet, only `TalkPageTopic`s